packages/pharos/tokens/assets/icons.json

```diff
@@ -36,7 +35,10 @@
       "email": { "value": "assets/icons/email.svg" },
       "exclamation-inverse": { "value": "assets/icons/exclamation-inverse.svg" },
       "exclamation": { "value": "assets/icons/exclamation.svg" },
-      "expand": { "value": "assets/icons/expand.svg" },
```

Comment:
If we're removing an icon and renaming another icon, these are both breaking changes. You can consider deprecating the existing icons for now, and they can be removed in a later major version release. We don't have a convention for deprecating tokens at the moment, but Style Dictionary offers a reasonable suggestion [here](https://github.com/amzn/style-dictionary/blob/main/examples/advanced/tokens-deprecation/tokens/color/base.json#L9-L13) that would look like this:

```json
"collapse": {
    "value": "assets/icons/collapse.svg",
    "deprecated": true,
    "deprecated_comment": "Use collapse-<direction> instead"
}
```